lib/oauth.js

```diff
@@ -179,7 +185,7 @@ module.exports = new Model({
             }
           } catch (error) {
             if (error.name === 'SecurityError') {
-              error = new Error('No existing Panoptes session found');
+              console.warn('No existing Panoptes session found');
```

Comment:
why not logging the actual `error` as well?

Reply:
That happens on the next line, when the promise rejects.